docs/pages/application-access/enroll-kubernetes-applications/reference.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Reference
+description: This guide is a comprehensive reference of configuration options for automatically enrolling Kubernetes applications with Teleport.
+---
+
+## Configuring Teleport agents
+
```

Comment:
@AntonAM can we add examples of how to configure these values in the helm chart.
I ask because APp discovery must be in cluster so the preferred way is the helm chart.